builds/e2e/nested-e2e.yaml

```diff
@@ -109,7 +109,6 @@ stages:
         - Agent.OS -equals Linux
         - Agent.OSArchitecture -equals X64
         - status -equals unlocked_$(Build.BuildId)
-        - $(agent.cleanIdentity)
```

Comment:
Why remove?

Reply:
I added for ease of debugging. Not sure if we want an unused var in the pipeline.